package.json

```diff
@@ -2,15 +2,18 @@
   "name": "discovery",
   "description": "App to view, filter and edit CDF data and relationships.",
   "version": "0.1.4",
-  "contributors": [ "David Liu <david.liu@cognite.com>", "Anders Hafreager <anders.hafreager@cognite.com>" ],
+  "contributors": [
```

Comment:
Why do I see this change in so many PR's?

Reply:
fixed!